scripts/babel-plugin-jest-require-outside-vm.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const assert = require('assert');
+
+/*
+Replace
+
+requireOutside('package')
+
+with
+
+require(require.resolve('package', {
+  [Symbol.for('jest-resolve-outside-vm-option')]: true,
+}));
+*/
+
+const REQUIRE_OUTSIDE_FUNCTION_NAME = 'requireOutside';
+
+module.exports = ({template, types: t}) => {
+  const replacement = template(`
+    require(require.resolve(IMPORT_PATH, {
+      [Symbol.for('jest-resolve-outside-vm-option')]: true,
```

Comment:
`Symbol` global might be wrong, maybe use https://github.com/facebook/jest/blob/dbb12903801f7423f0273d4c1872294e9dbbc5ce/scripts/babel-plugin-jest-native-globals.js#L17-L19 somehow?

Reply:
Ok. I'm gonna inline it though, don't want to bother with 'what if `Symbol` is already declared in the same scope